openpype/modules/deadline/plugins/publish/submit_max_deadline.py

```diff
@@ -49,11 +49,13 @@ def get_job_info(self):
 
         instance = self._instance
         context = instance.context
-
         # Always use the original work file name for the Job name even when
         # rendering is done from the published Work File. The original work
         # file name is clearer because it can also have subversion strings,
         # etc. which are stripped for the published file.
+        if not instance.data.get("publish"):
+            self.use_published = False
```

Comment:
This doesn't sound right. If the instance itself has `publish` set to something that equates to False than the instance shouldn't get processed to begin with.